include/groonga/groonga.h

```diff
@@ -1343,6 +1343,15 @@ grn_obj_unlock(grn_ctx *ctx, grn_obj *obj, grn_id id);
  */
 GRN_API grn_rc
 grn_obj_clear_lock(grn_ctx *ctx, grn_obj *obj);
+/**
+ * \brief Check if an object is currently locked.
+ *
+ * \param ctx The context object.
+ * \param obj The target object whose lock status is to be checked.
+ *
+ * \return 0 if there are no locks. Return a non-zero unsigned integer
```

Comment:
```suggestion
 * \return `0` if there are no locks. Return a non-zero unsigned integer
```

数値は ` で囲っているので合わせると良いです。

Reply:
fix: [de705a6](https://github.com/groonga/groonga/pull/1995/commits/de705a6ceb5dcef00b35fff32901b6914bb122a8)